.gitignore

```diff
@@ -0,0 +1,50 @@
+#REACT
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/react-time-track/node_modules
+/react-time-track/.pnp
+.pnp.js
+
+# testing
+/react-time-track/coverage
+
+# production
+/react-time-track/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+
+#RAILS
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
```

Comment:
Hi  Jonathan, do we need to add "/rails-time-track/" to this line? Or it is fine as is?